From 0f0601e79f9ce7614d157284523e6cd8af2259d5 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Wed, 5 Jan 2022 12:12:47 +0100
Subject: [PATCH] numpy/core/setup.py: disable svml for now

The check really doesn't work in cross compiling,
as it is using host python to see what arch we're
building on. Issue reported upstream:
https://github.com/numpy/numpy/issues/20736

Upstream-Status: Inappropriate [needs upstream fix]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 numpy/core/setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/numpy/core/setup.py b/numpy/core/setup.py
index a5f423d..945f2dd 100644
--- a/numpy/core/setup.py
+++ b/numpy/core/setup.py
@@ -70,7 +70,7 @@ def can_link_svml():
     """
     machine = platform.machine()
     system = platform.system()
-    return "x86_64" in machine and system == "Linux"
+    return False
 
 def check_svml_submodule(svmlpath):
     if not os.path.exists(svmlpath + "/README.md"):
